### DISCUSSION TITLE
Public event portal

### DISCUSSION THREAD

**Discussion by johnmeshulam** [FIRSTIsrael/lems#752] [Ideas]
## Why?

We want tams to be able to view public information through a subdomain of LEMS.

### What?

- Queueing status (Similat to FRC's Nexus)
- Schedule
- Scoreboard
- Event results (once complete)
- Team list (names numbers affiliations)
- Scorer (Local state only, no saving in DB)

**Comment by itamaroryan** [comment]
Closing as this was completed in #866
